src/libasr/pass/intrinsic_function_registry.h

```diff
@@ -76,6 +76,7 @@ enum class IntrinsicScalarFunctions : int64_t {
     SymbolicLog,
     SymbolicExp,
     SymbolicAbs,
+    SymbolicHasQ,
```

Comment:
Let's call it:
```suggestion
    SymbolicHasSymbolQ,
```
I think that is more specific to what it does.

Reply:
Ohh so aren't we fully recreating what SymPy does? As in SymPy's `has` function checks not only for Symbols but also for subexpressions. Isn't that not expected out of the intrinsic fucntion ?

Reply:
We have to start with what symengine allows. Why don't you submit the PR to symengine, let's get it merged there, make a release into conda and finish this PR?

Reply:
Yeah I just forked symengine, I've made the change but I haven't contributed to symengine before hence I am not sure how do I test my change out. Looking into it !!

Reply:
I've raised a PR here but I am somewhat skeptical with the implementation https://github.com/symengine/symengine/pull/1978
You could guide me further.